quickstart/gcp/setup.md

```diff
@@ -31,10 +31,18 @@ To communicate your credentials to the Pulumi Google Cloud Platform Provider,
 export the contents of your credentials file to the `GOOGLE_CREDENTIALS`
 environment variable:
 
+Linux and Mac OS X
+
 ```bash
 export GOOGLE_CREDENTIALS=$(cat credentials.json)
 ```
 
+Windows Powershell
+
+```bash
+$env:GOOGLE_CREDENTIALS="C:\PATH\TO\CREDENTIALS\credentials.json"
+```
```

Comment:
Does this do the same thing as the `bash` above?  In particular, does it include the contents of the file as the value of the `GOOGLE_CREDENTIALS` environment variable - or does it set the name of the file as the value of the environment variable?